test/sqllogictest/tpch_materialized_view.slt

```diff
@@ -0,0 +1,1891 @@
+                    Distinct group_by=[p_partkey] // { arity: 1 }
+                      Project (#0) // { arity: 1 }
+                        Get l4 // { arity: 9 }
+                  Get l1 // { arity: 5 }
+                  Get l0 // { arity: 7 }
+                  Get l2 // { arity: 4 }
+                  Get l3 // { arity: 3 }
+  With
+    cte l4 =
+      Project (#0, #2, #10, #11, #13..=#15, #19, #22) // { arity: 9 }
+        Filter (p_size = 15) AND (r_name = "EUROPE") AND (p_partkey) IS NOT NULL AND (s_suppkey) IS NOT NULL AND (s_nationkey) IS NOT NULL AND (n_regionkey) IS NOT NULL AND like["%BRASS"](varchar_to_text(p_type)) // { arity: 28 }
+          Join on=(p_partkey = ps_partkey AND s_suppkey = ps_suppkey AND s_nationkey = n_nationkey AND n_regionkey = r_regionkey) type=delta // { arity: 28 }
+            implementation
+              %0:part » %2:l1[#0]KA » %1:l0[#0]KA » %3:l2[#0]KA » %4:l3[#0]KAef
+              %1:l0 » %2:l1[#1]KA » %0:part[#0]KAelf » %3:l2[#0]KA » %4:l3[#0]KAef
+              %2:l1 » %0:part[#0]KAelf » %1:l0[#0]KA » %3:l2[#0]KA » %4:l3[#0]KAef
+              %3:l2 » %4:l3[#0]KAef » %1:l0[#3]KA » %2:l1[#1]KA » %0:part[#0]KAelf
+              %4:l3 » %3:l2[#2]KA » %1:l0[#3]KA » %2:l1[#1]KA » %0:part[#0]KAelf
+            ArrangeBy keys=[[p_partkey]] // { arity: 9 }
+              ReadIndex on=part pk_part_partkey=[delta join 1st input (full scan)] // { arity: 9 }
+            Get l0 // { arity: 7 }
+            Get l1 // { arity: 5 }
+            Get l2 // { arity: 4 }
+            Get l3 // { arity: 3 }
+    cte l3 =
+      ArrangeBy keys=[[r_regionkey]] // { arity: 3 }
+        ReadIndex on=region pk_region_regionkey=[delta join lookup] // { arity: 3 }
+    cte l2 =
+      ArrangeBy keys=[[n_nationkey], [n_regionkey]] // { arity: 4 }
+        ReadIndex on=nation pk_nation_nationkey=[delta join lookup] fk_nation_regionkey=[delta join lookup] // { arity: 4 }
+    cte l1 =
+      ArrangeBy keys=[[ps_partkey], [ps_suppkey]] // { arity: 5 }
+        ReadIndex on=partsupp fk_partsupp_partkey=[delta join lookup] fk_partsupp_suppkey=[delta join lookup] // { arity: 5 }
+    cte l0 =
+      ArrangeBy keys=[[s_suppkey], [s_nationkey]] // { arity: 7 }
+        ReadIndex on=supplier pk_supplier_suppkey=[delta join lookup] fk_supplier_nationkey=[delta join lookup] // { arity: 7 }
+
+Used Indexes:
+  - materialize.public.pk_nation_nationkey (delta join lookup)
+  - materialize.public.fk_nation_regionkey (delta join lookup)
+  - materialize.public.pk_region_regionkey (delta join lookup)
+  - materialize.public.pk_part_partkey (delta join 1st input (full scan))
+  - materialize.public.pk_supplier_suppkey (delta join lookup)
+  - materialize.public.fk_supplier_nationkey (delta join lookup)
+  - materialize.public.fk_partsupp_partkey (delta join lookup)
+  - materialize.public.fk_partsupp_suppkey (delta join lookup)
+
+EOF
+
+
+query T multiline
+-- Query 03
+EXPLAIN WITH(humanized_exprs, arity, join_impls)
```

Comment:
Do we want to document humanized_exprs by the way? I didn't know about it before.

Reply:
> Do we want to document humanized_exprs by the way? I didn't know about it before.

This landed last week and has not yet made it's way to production. I have a reminder to put together an announcement tomorrow and possibly update the docs.